src/facade/dragonfly_connection.cc

```diff
@@ -263,21 +391,30 @@ void Connection::RegisterOnBreak(BreakerCb breaker_cb) {
   breaker_cb_ = breaker_cb;
 }
 
-void Connection::SendMsgVecAsync(const PubMessage& pub_msg, fibers_ext::BlockingCounter bc) {
+void Connection::SendMonitorMsg(std::string_view monitor_msg,
+                                util::fibers_ext::BlockingCounter bc) {
```

Comment:
why do you pass "bc" all the way from MonitorCmd to here?

Reply:
This is the same way we are doing it in the pub/sub support. This sync all the used connection to the one location where it is used. Maybe we don't need this in this case? let talk about this directly?